events/events-team/registration/README.md

```diff
@@ -162,37 +163,74 @@ yq '.admins + .members' \
   | sed -e 's/"//g' > members.txt
 ```
 
+an alternative is 
+
+```
+git clone https://github.com/kubernetes/org.git
+cd org
+yq '.admins + .members' \
+  config/etcd-io/org.yaml \
+  config/kubernetes/org.yaml \
+  config/kubernetes-client/org.yaml \
+  config/kubernetes-csi/org.yaml \
+  config/kubernetes-sigs/org.yaml \
+  | sed -e 's/---//g' | sed -e 's/- //g' \
+  | sed -e 's/"//g' | sed -e '/^$/d' \
+  | awk '{print tolower($0)}' | sort | uniq > members.txt
+```
 
+Populate what's in members.txt in the **org - DO NOT DELTE** tab in the 
+**Kubernetes Contributor Summit Planning Doc**.
 
-An approval process for registration is commonly used for multiple reasons, but
-really boils down to ensuring the right people are in the right track or room.
-The goal is for all contributors to get the best experience out of the event and
-that may mean asking them to switch tracks.
+Create a list of staff, shadows, volunteers including day-of volunteers. You can 
+create this list from the GitHub staffing issue e.g. https://github.com/kubernetes/community/issues/7611
+and query the Summit Staff leads for their day-of volunteers. The Day-of Operations
+team will likely have volunteers not listed on the GitHub staffing issue.
 
-A few scenarios that have occurred:
+Once the talks for the Kubernetes Contributor Summit have been selected, create 
+a list of speakers and note if any are not org members so they can be approved
+when they register.
 
-**A person has signed up for the New Contributor Workshop, but in their
-comments have mentioned they are unfamiliar with Kubernetes.**
+Review past declined registrants if any are speakers, email them to re-register 
+for the Contributor Summit.
+
+#### Exception Request Scenarios
+
+**A person is not an org member and would like to attend the Contributor Summit**
+
+With 1600+ Kubernetes Org members and limited space and budget requirements, we've
+made the decision to limit attendance to those that are already actively engaged
+with the project. 
+
+Contributor Summit speakers without org membership can attend the Contributor Summit.
+
+A SIG lead (co-chair, tech lead, subproject lead) can sponsor a non-org member to attend the summit,
```

Comment:
is there a limit on the number of non-members a SIG lead can sponsor?

This doesnt have to be addressed in here. When the policy for sponsorship is written, it might be good to mention the limit if there is one

Reply:
Currently, there is no limit -- not that I'm aware of